docs/Installation.md

```diff
@@ -23,74 +24,69 @@ Since we use the Symfony framework, the [minimal requirements of Symfony][2] app
 
 Apart from these you will also need the following :
 
-- [Node.js][3]
-- [Sass][4]
-- [Bower][5]
-- [Grunt][6]
+- [Node.js][4]
+- [Sass][5]
+- [Bower][6]
+- [Grunt][7]
 - UglifyJS
 - UglifyCss
-- a database server, preferrably [MySQL][7] 5.x (we haven't tried any other DB yet...)
+- a database server, preferrably [MySQL][8] 5.x (we haven't tried any other DB yet...)
 - a web server...
 
 To install the required dependencies (apart from the database and web server) :
 
-- On OS X (using [Homebrew][8]):
+ - On OS X (using [Homebrew][9]):
 
-```
-brew install node
-gem install sass # maybe you need sudo...
-npm install -g bower
-npm install -g grunt
-npm install -g grunt-cli
-npm install -g uglify-js
-npm install -g uglifycss
-```
+   ```
+   brew install node
+   gem install sass # maybe you need sudo...
+   npm install -g bower
+   npm install -g grunt
+   npm install -g grunt-cli
+   npm install -g uglify-js
+   npm install -g uglifycss
+   ```
 
-- On Linux ([Debian][9] / [Ubuntu][10]):
+ - On Linux ([Debian][10] / [Ubuntu][11]):
 
-```
-sudo apt-get install node
-gem install sass # maybe you need sudo...
-npm install -g bower
-npm install -g grunt
-npm install -g grunt-cli
-npm install -g uglify-js
-npm install -g uglifycss
-```
+   ```
+   sudo apt-get install node
+   gem install sass # maybe you need sudo...
+   npm install -g bower
+   npm install -g grunt
+   npm install -g grunt-cli
+   npm install -g uglify-js
+   npm install -g uglifycss
+   ```
 
-- On Windows:
+ - On Windows:
 
-```
-We kindly suggest installing [VirtualBox][11], [VMware][12] or another virtualization platform and running a Linux
-VM on it, or basically : you're on your own.
-```
+   We kindly suggest installing [VirtualBox][12], [VMware][13] or another virtualization platform and running a Linux
```

Comment:
This should not be rendered in a code block, like it was before. The indentation is 3 spaces to be outlined with the list item, so it's rendered inside the list item.